lib/revisement/changeset/file_modification.rb

```diff
@@ -97,8 +99,8 @@ def yaml_diff(keypath)
         return unless (diff = Xcodeproj::Differ.diff(
           prior,
           current,
-          key_1: 'prior_revision',
-          key_2: 'current_revision'
+          key_1: 'prior_revision', # rubocop:disable Naming/VariableNumber
```

Comment:
Thoughts about fixing with `key_one` and `key_two` instead of disabling the lint check here?

Reply:
I actually don't think I can, unless I want to start digging into `Xcodeproj::Differ`:

```ruby
---
a:
-  prior_revision: bbcc
-  current_revision: bc
+  value_1: bbcc
+  value_2: bc
```

Changing the keys mangles the output from the diffed YAML. I'm going to wrap this in a disable at least to try and limit the blast radius of these changes.